openfisca_us/variables/ssa/ssi/ssi.py

```diff
@@ -3,15 +3,21 @@
 
 class ssi(Variable):
     value_type = float
-    entity = SPMUnit
+    entity = Person
     definition_period = YEAR
-    documentation = "Supplemental Security Income amount"
+    documentation = "Supplemental Security Income"
     label = "Supplemental Security Income"
     unit = USD
 
-    def formula(spm_unit, period, parameters):
-        # Obtain eligibility.
-        eligible = spm_unit("is_ssi_eligible", period)
-        # Obtain amount they would receive if they were eligible.
-        amount_if_eligible = spm_unit("ssi_amount_if_eligible", period)
-        return where(eligible, amount_if_eligible, 0)
+    def formula(person, period, parameters):
+        abd = person("is_ssi_aged_blind_disabled", period)
+        countable_resources = person("ssi_countable_resources", period)
+        p_ssi = parameters(period).ssa.ssi
+        # Only individual is modeled currently.
+        resource_limit = p_ssi.eligibility.resources.limit.individual
+        meets_resource_test = countable_resources <= resource_limit
+        # Calculate amount.
+        amount = parameters(period).ssi.amount.individual
```

Comment:
The error message obfuscated this, but here's the issue:
```suggestion
        amount = parameters(period).ssa.ssi.amount.individual
```